### PR TITLE
Update universite-du-quebec-a-montreal-departement-dhistoire.csl

### PR DESCRIPTION
**Pull request by McKaulick**
Fix chapter pages info

### CODE REVIEW

File: universite-du-quebec-a-montreal-departement-dhistoire.csl [hunk 1]
```diff
@@ -164,24 +164,22 @@
...
             <text variable="title" text-case="capitalize-first" quotes="true"/>
           </group>
-          <text value="dans" font-style="normal" suffix=" " prefix=" "/>
+          <text value="dans" font-style="normal" suffix="" prefix=" "/>
           <text variable="edition" suffix=" (dir.), "/>
           <text variable="container-title" text-case="capitalize-first" font-style="italic" prefix="&#160;"/>
         </group>
       </else-if>
       <else-if type="webpage" match="any">
-        <group delimiter=", ">
```
Comment (adam3smith) [inline universite-du-quebec-a-montreal-departement-dhistoire.csl:173]: Why this change? -- moving from group delimiters to suffixes is the opposite of what we recommend for robus styles (it'll make little difference here,  but there are still edge cases where it might matter.

Comment (McKaulick) [reply universite-du-quebec-a-montreal-departement-dhistoire.csl:173]: Hello Adam, you're right. I changed the code. 

Thank you,

**Comment by adam3smith** [comment]
Thanks!
